src/components/uploadPicture/uploadImage.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState, useEffect, useCallback } from "react";
+import styles from "./uploadImage.module.css";
+import { trpc } from "@/client/lib/trpc";
+import axios from "axios";
+import CameraIcon from "@mui/icons-material/PhotoCamera";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+
+interface UploadImageComponentProps {
+  title: string;
+  fileName: string;
+  onUpload: (fileId: string) => void;
+}
+
+const UploadImageComponent: React.FC<UploadImageComponentProps> = ({
+  title,
+  fileName,
+  onUpload,
+}) => {
+  const [showStart, setShowStart] = useState(true);
+  const [showTakePicture, setShowTakePicture] = useState(false);
+  const [showUploadPicture, setShowUploadPicture] = useState(false);
+  const [startButtonDisabled, setStartButtonDisabled] = useState(false);
+  const [uploading, setUploading] = useState(false);
```

Comment:
I don't think this variable is necessary
it slightly obfuscates the logic flow since control now goes through a `useEffect()`